### PR TITLE
Create News “2018-07-11-navcoin-core-4-3-0-release-candidate”

### PR DESCRIPTION
**Pull request by craigmacgregor**
Automatically generated by Netlify CMS

### CODE REVIEW

File: content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: NavCoin Core 4.3.0 Release Candidate
+author: Craig MacGregor
+date: '2018-07-11T19:11:27+12:00'
+feature_image: /images/uploads/11-07-2018.png
+categories:
+  - Nav News
+---
+This week brings your chance to test and comment on the release candidate for NavCoin Core 4.3.0. This update introduces the OpenAlias feature for NavCoin core, adds wallet support for bootstrapping, and delivers some GUI fixes.
+
+This week brings your chance to test and comment on the release candidate for NavCoin Core 4.3.0. This update introduces the OpenAlias feature for NavCoin core, adds wallet support for bootstrapping, and delivers some GUI fixes.
```
Comment (IfaqYurmama) [inline content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md]: is it intended that this paragraph is duplicated?

File: content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Check out the pull request in GitHub here:
+
+<https://github.com/NAVCoin/navcoin-core/pull/232>
```
Comment (IfaqYurmama) [inline content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md]: I'd move this behind the first paragraph at least or to the very end

File: content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md (hunk 1 above)
Comment (craigmacgregor) [reply content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md]: Yeah, for some reason the templating engine pulls out the first paragraph as the text for the /news page and omits it from the main article page, so we have to duplicate it like this so it displays in both locations.

Comment (craigmacgregor) [reply content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md]: if you check out the preview, it looks fine, it's not duplicated on the article page.

File: content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md (hunk 2 above)
Comment (craigmacgregor) [reply content/news/2018-07-11-navcoin-core-4-3-0-release-candidate.md]: I think the guys really want people to engage with the PR which is why it's at the top like that.

**Pull request merged**